Covid-19 food supply App (Easy) Solution

Vanshikaa00 opened the pull request:
Solution for covid 19 food supply app problem solved in java

Vanshikaa00 committed:
Covid-19 food supply App (Easy)

Solution for covid 19 food supply app problem solved in java

irshadjsr21 reviewed (changes requested):
Please send a PR to `girlscript-blr:solution` branch. Kindly refer to [CONTRIBUTING.md](https://github.com/girlscript-blr/code-with-girlscript-bangalore/blob/master/CONTRIBUTING.md) file's Create Pull Request point.

Now that you have created the PR, you can click on the `Edit` button beside the PR title and change the branch this PR is pointing to. Please follow the below steps.

1. Click on `Edit`.
![image](https://user-images.githubusercontent.com/40335357/86520924-8f1a5d00-be67-11ea-87e9-b4b1d9d57b98.png)

2. Change the base branch from `master` to `solution`.
![image](https://user-images.githubusercontent.com/40335357/86520931-a3f6f080-be67-11ea-85e3-9b57690e483c.png)

irshadjsr21 reviewed (changes requested):
Please update your branch with the latest `master` branch. To do this follow these steps:

- `git fetch --all`
- `git rebase upstream/master`
- Resolve the conflicts if any

Vanshikaa00 committed:
update2

irshadjsr21 commented on Easy/3. COVID-19 Food Supply App/solutions/README.md:
```diff
@@ -1 +1,92 @@
-# Solutions
+
```
Please revert all the changes made to this file. This PR should contain only the changes to your folder, i.e., `Vanshikaa00_Vanshika` folder.

Vanshikaa00 replied:
Hi @irshadjsr21 i followed the above steps you mentioned to update my branch foodSupplyEasy and also made the changes you requested in README.md file of solutions folder but when i try to push the changes made in  foodSupplyEasy branch (commit: update 2), i am receiving errors and if i try the hints too, i still get an error . I tried to search for a solution on internet but that did'nt worked 😅
![Screenshot (1490)](https://user-images.githubusercontent.com/53425575/93779150-48f1c000-fc44-11ea-8854-458809177137.png)

irshadjsr21 replied:
If your local repo has the right code, you can try `git push -f -u origin foodSupplyEasy`

Vanshikaa00 committed:
update 2

reverted all the changes made to README.md file in Solutions folder

irshadjsr21 commented on Easy/3. COVID-19 Food Supply App/README.md:
```diff
@@ -87,4 +87,6 @@ Follow the steps mentioned in [this](../../CONTRIBUTING.md) file to submit your
 Solved this problem? Then you might want to checkout the other versions of this problem.
 
 - [Medium](../../Medium/3.%20COVID-19%20Food%20Supply%20App/README.md)
+
```
Please revert all the changes shown in the files other than your folder.

Vanshikaa00 committed:
update 3

irshadjsr21 commented:
Please make the necessary changes by 1st week of October for this PR to be approved.

Vanshikaa00 commented:
Hey @irshadjsr21 ,I already made the requested changes in the above commit named "update 3" a few days back and I don't know where things are going wrong as i reverted back all of the changes in README.md files as you requested. Actually my forked repo also had some push errors which i addressed above and i can't quite figure out what's going wrong😅. Can i send a new PR for the same challenge again?

irshadjsr21 commented:
Hi @Vanshikaa00 , You can go to the `Files Changed` tab of this PR and see what all changes have been done. You'll see the 2 Readme.md files that have been changed. You need to revert those changes which are not in your folder.

Please let me know if this is clear to you.

Vanshikaa00 committed:
update 4

changed spacing in readme files

Vanshikaa00 commented:
made the changes in commit "update 4" @irshadjsr21

irshadjsr21 reviewed (approved):
I'm really sorry, I missed the notification for your latest changes to this PR.

The solution is correct and we'll make sure this PR counts.

irshadjsr21 commented:
Greetings from Code With Girlscript Bangalore😃

Dear participant, we have gone through your pull request made in `EASY` sections of the problem statement.

Congratulations🎊 🎉  on getting your pull request successfully merged in the **COVID-19 Food Supply App - Easy** section in **code-with-girlscript-bangalore** repository. We will be distributing your valuable badges🥇🥈🥉🎖️ corresponding to the pull requests that are merged.

Learning is a never-ending process. 📈 Check out the October problem statement and try your hand at it for earning higher-level badges. Thanks for your contribution! Happy coding😃

Vanshikaa00 commented:
Oh.. It's alright 😄 and thank you for merging 👍
